script/install-deps

```diff
@@ -29,25 +29,79 @@
 
 . "$(dirname $0)"/common.sh
 
+set -e
+
+apt_update_once=0
+
+install_package() {
+    local cmd=$1
+    shift 1
+
+    # only install if command not available
+    if installed "$cmd"; then
+        return 0
+    fi
+
+    while (("$#")); do
+        case "$1" in
+        --apt)
+            if installed apt-get; then
+                if [ "$apt_update_once" -eq "0" ]; then
+                    sudo apt-get update || die "apt-get update failed"
+                    apt_update_once=1
+                fi
+
+                sudo apt-get install -y --no-install-recommends "$apt_pkg"
+                return 0
+            fi
+
+            shift 2
+            ;;
+        --brew)
+
+            if installed brew; then
+                brew install "$brew_pkg"
+                return 0
+            fi
+
+            shift 2
+            ;;
+        *)
+            PARAMS="$PARAMS $1"
+            echo "Error: Unsupported flag $1" >&2
+            return 1
+            ;;
+        esac
+    done
+
+    die "Can not install $cmd on your system, please retry after installing $cmd."
```

Comment:
The error message doesn't make sense to me.

Reply:
fixed